build/embed/fluent-bit.version

```diff
@@ -6,6 +6,6 @@
 # OS, newrelic_plugin_version, fluent-bit
 
 linux,1.19.1
-windows,1.19.1,3.0.6
+windows,1.19.1,3.1.9
```

Comment:
have you done any testing also?

Reply:
No , ci workflows does the testing . slack thread reg: https://newrelic.slack.com/archives/C07AFGCA2D9/p1729504502729789?thread_ts=1729242572.920869&cid=C07AFGCA2D9